docs-requirements.txt

```diff
@@ -3,3 +3,4 @@
 -r test-requirements.txt
 sphinx~=1.4.9
 sphinx-argparse~=0.1.15
+sphinx_rtd_theme
```

Comment:
@satwikkansal Yeah I guess that info is outdated.
docs-requirements.txt is all you need :)
Thanks!